agents/utils/schema.py

```
@@ -0,0 +1,51 @@
+from enum import Enum
+
+from cdp import Wallet
```

Ruff (0.8.2)

`cdp.Wallet` imported but unused

Remove unused import: `cdp.Wallet`

(F401)